testing/citest/spinnaker_testing/gcs_pubsub_trigger_agent.py

```diff
@@ -121,8 +121,14 @@ def detail(self):
 
   @property
   def timed_out(self):
+    ping = datetime.datetime.utcnow()
+    diff = ping - self.__start
+    return diff > datetime.timedelta(minutes=1)
```

Comment:
Please make this a constant up in the class.
Better yet would be a property on the agent that you coudl initialize as 1, even if you dont provide a setter at this time.

Reply:
Done.